File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def abs(a:int):
        if a>0:
            return a
        else:
            return -a
    def differenceOfSum(self, nums: List[int]) -> int:
        s,ds=0,0
        for i in nums:
            s+=i
            x=i
            while x>0:
                ds+=x%10
                x=x//10
        return abs(s-ds)
            

File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.rs
impl Solution {
    pub fn abs(n:i32)-> i32{
        if n>0{
            n
        }else {
            -n
        }
    }
    pub fn difference_of_sum(nums: Vec<i32>) -> i32 {
        let mut s=0;
        let mut ds=0;
        for i in nums {
            let mut x = i;
            s += i;
            while x>0 {
                ds += x%10;
                x /= 10;
            }
        }
        Solution::abs(s-ds)
    }
}